app/views/shared/_order_detail_cell.html.haml

```diff
@@ -10,11 +10,12 @@
     - if od.product.offline?
       = tooltip_icon "icon-warning-sign", t("instruments.offline.note")
 
-  - if od.survey_completed?
-    .order-detail-extra= link_to 'View Order Form', od.external_service_receiver.show_url, :target => '_blank'
-  - od.stored_files.template_result.each do |stored_file|
-    .order-detail-extra
-      = link_to t("order_details.show.link.view.order_file"), facility_template_result_path(stored_file)
+  - if can?(:template_results, od)
+    - if od.survey_completed?
+      .order-detail-extra= link_to 'View Order Form', od.external_service_receiver.show_url, target: '_blank'
```

Comment:
Looks like lint didn't pick up the single quotes. I wouldn't mind I18n this as well.